src/LessMsi.Gui/MainFormPresenter.cs

```diff
@@ -45,6 +47,7 @@ class MainFormPresenter
 	{
 		private readonly MainForm _view;
         private SortableBindingList<MsiFileItemView> fileDataSource;
+		private IList<object[]> tableViewDataSource;

```

Comment:
This file (and many others) have mixed tabs / spaces.
I assumed that tabs were the new 'default', and added a local `.editorconfig` file to enforce that.